helm/provisioner/tests/chart_test.go

```diff
@@ -52,6 +52,18 @@ var testCases = []testCase{
 		name:          "provisionerd_psk",
 		expectedError: "",
 	},
+	{
+		name:          "provisionerd_key",
+		expectedError: "",
+	},
+	{
+		name:          "provisionerd_psk_and_key",
```

Comment:
Is it ever sensible to accept both? I think we can only accept one or other as the authentication credential